next.config.js

```diff
@@ -1,10 +1,7 @@
 const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true',
 })
-const withSass = require('@zeit/next-sass')
```

Comment:
Should this package also be removed from package.json`?

Reply:
Yeah, it has been removed
https://github.com/dddwa/dddperth-website/pull/264/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L51